src/ai/backend/agent/config/unified.py

```diff
@@ -994,14 +1034,8 @@ class DockerExtraConfig(BaseConfigSchema):
     )
 
 
-class AgentUnifiedConfig(BaseConfigSchema):
+class AgentGlobalConfig(BaseConfigSchema):
```

Comment:
It would be good if docstrings were added overall, since several Agent-related configs have been created.